code/game/gamemodes/umbrage/umbrage_abilities/devour_will.dm

```diff
@@ -0,0 +1,30 @@
+//After a brief charge-up, equips a temporary dark bead that can be used on a human to knock them out and drain their will, making them vulnerable to conversion.
+/datum/action/innate/umbrage/devour_will
+	name = "Devour Will"
+	id = "devour_will"
+	desc = "Creates a dark bead that can be used on a human to fully recharge psi and knock them out."
+	button_icon_state = "umbrage_devour_will"
+	check_flags = AB_CHECK_RESTRAINED|AB_CHECK_STUNNED|AB_CHECK_LYING|AB_CHECK_CONSCIOUS
+	psi_cost = 20
+	lucidity_cost = 0 //Baseline
+	blacklisted = 1
+	var/list/victims = list() //A list of people we've used the bead on recently; we can't drain them again so soon
```

Comment:
Oh. Right.

Why does Byond have hidden procs anyway? A better question is, how the hell do you people manage to find them?